### DISCUSSION TITLE
Documentation Site "API Documentation" links are no longer correct.

### DISCUSSION THREAD

**Discussion by CelDaemon** [libjpeg-turbo/libjpeg-turbo#788] [Developer Discussion]
I didn't think this was worthy of being made an issue, as it's not an issue with the library itself.
However, when I was taking a look at the site to see if I could use this library, I noticed that the links were dead.

For example:

- The `usage.txt` link points to [`/main/usage.txt`](https://raw.githubusercontent.com/libjpeg-turbo/libjpeg-turbo/main/usage.txt), meanwhile the file seems to have been moved to `/doc/usage.txt`
- The `libjpeg.txt` link is incorrect in the same way as usage.txt is.
- The `example.c` file is also wrong, however it seems to have been moved to `/src/example.c` instead.

**Comment by dcommander** [comment]
Just an oversight from merging the new directory structure into main in preparation for the 3.1 beta release earlier today. I'll fix it tomorrow.
